hvpy/__init__.py

```diff
@@ -1 +1,2 @@
+from .facade import getJP2Image
```

Comment:
This is ok for now, but may change to `import *` in the future when we have a bunch of API functions.

Reply:
I think this should be done now.

Reply:
Should I mark this as resolved?